ci-operator/step-registry/openshift/e2e/aws/metadata-service-auth-required/openshift-e2e-aws-metadata-service-auth-required-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: openshift-e2e-aws-metadata-service-auth-required
+  steps:
+    pre:
+      - chain: ipi-conf-aws
+      - ref: ipi-conf-aws-metadata-service-auth
+      - chain: ipi-install
```

Comment:
The existing ipi-aws-pre chain already [contains a bunch of environment knobs][1].  It's unclear to me why you want a new workflow for this knob, vs. adding the knob to the existing chain and then setting the `env` you want in your job configuration.  Using a workload to centralize config between multiple jobs makes sense for flavors that have lots of consuming jobs, but you only have the two consuming jobs...

[1]: https://steps.ci.openshift.org/chain/ipi-aws-pre

Reply:
Hm, I can not clearly explain the reasoning, why i did exactly that there. Thought that it should be the right thing. :D 

Would it be better to extend [ipi-conf-aws](https://steps.ci.openshift.org/reference/ipi-conf-aws) instead and expose extra knob there?

Reply:
Touching the main workload will indeed create a bunch of rehearsals on this one release PR.  But how often do we expect to have to touch this logic in the release repo?  I expect it will be rare, and the maintenance overhead of having devs keeping track of more decoupled pieces isn't free either :wink:

I'm not all that concerned either way though, so :shrug: whatever makes the most sense to the folks who will be maintaining this code.

Reply:
https://github.com/openshift/release/compare/master...lobziik:imdsv2-install-aws-ipi-conf?expand=1 - i prepared patch which adds env knob to `ipi-conf-aws` step. Would like to get a review before i will update this pr... Still a bit afraid of massive rehearsals. Can you please check if it looks sane? :)